website/pages/docs/k8s/crds/upgrade-to-crds.mdx

```diff
@@ -0,0 +1,318 @@
+   ```
+1. Now you're ready to create a custom resource that takes over control of this
+   config entry. The custom resource will look like:
+
+   ```yaml
+   apiVersion: consul.hashicorp.com/v1alpha1
+   kind: ServiceDefaults
+   metadata:
+     name: static-client
+   spec:
+     protocol: 'http'
+   ```
+
+   Where `metadata.name` is the name of your service and `spec.protocol` is
+   the default protocol you've set.
+
+1. When you run `kubectl apply` on this file, the `ServiceDefaults` custom
+   resource should be created successfully and its `synced` status will be `True`:
+
+   ```shell-session
+   $ cat <<EOF | kubectl apply -f -
+   apiVersion: consul.hashicorp.com/v1alpha1
+   kind: ServiceDefaults
+   metadata:
+     name: static-client
+   spec:
+     protocol: "http"
+   EOF
+   servicedefaults.consul.hashicorp.com/static-client created
+
+   $ kubectl get servicedefaults static-client
+   NAME            SYNCED   AGE
+   static-client   True     1s
+   ```
+
+1. Repeat steps 2-6 for each service-defaults config entry.
+1. You can now remove the `consul.hashicorp.com/connect-service-protocol`
+   annotation from your deployments to indicate it is no longer used.
+
+## Proxy Defaults
+
+If you are setting `connectInject.centralConfig.proxyDefaults` then you must
+perform the follow steps to migrate to custom resources.
+
+1. Get your existing proxy-defaults config:
+   ```shell-session
+   $ consul config read -name global -kind proxy-defaults
+   {
+       "Kind": "proxy-defaults",
+       "Name": "global",
+       "Config": {
+           "local_connect_timeout_ms": 1000
+       },
+       "MeshGateway": {
+           "Mode": "local"
+       },
+       "Expose": {},
+       "CreateIndex": 4,
+       "ModifyIndex": 4
+   }
+   ```
+1. Export to a file:
+
+   ```shell-session
+   $ consul config read -name global -kind proxy-defaults > proxy-defaults.json
+   ```
+
+1. Edit the file and add the key `"Meta": {"consul.hashicorp.com/source-datacenter": "dc1"}`.
+   Where `dc1` is the name of your datacenter. Make sure you add any missing trailing commas required for JSON:
+   ```json
+   {
+     "Kind": "proxy-defaults",
+     "Name": "global",
+     "Config": {
+       "local_connect_timeout_ms": 1000
+     },
+     "MeshGateway": {
+       "Mode": "local"
+     },
+     "Expose": {},
+     "CreateIndex": 4,
+     "ModifyIndex": 4,
+     "Meta": { "consul.hashicorp.com/source-datacenter": "dc1" }
+   }
+   ```
+1. Write the updated config entry:
+   ```shell-session
+   $ consul config write proxy-defaults.json
+   Config entry written: proxy-defaults/global
+   ```
+1. Now you're ready to create a custom resource that takes over control of this
+   config entry. The custom resource will look like:
+
+   ```yaml
+   apiVersion: consul.hashicorp.com/v1alpha1
+   kind: ProxyDefaults
+   metadata:
+     name: global
+   spec:
+     config:
+       local_connect_timeout_ms: 1000
+     meshGateway:
+       mode: local
+   ```
+
```

Comment:
Probably worth calling out that the `metadata.name` has to be `global`

Reply:
Added "Also, `metadata.name` must be `global`."